static/build/miyoo/app/lang/sv.lang

```diff
@@ -0,0 +1,135 @@
+	"4":	"NEOGEO",
+	"5":	"FC",
+	"6":	"SFC",
+	"7":	"GB",
+	"8":	"GBA",
+	"9":	"MD",
+	"10":	"GPSP",
+	"11":	"SMS",
+	"12":	"PS",
+	"13":	"PCE",
+	"14":	"ÖVRIGT",
+	"15":	"Inställningar",
+	"16":	"SPELINFO",
+	"17":	"ENHETSINFO",
+	"18":	"Nyligen",
+	"19":	"FIL",
+	"20":	"ADC-INFO",
+	"21":	"EncDec-Test",
+	"22":	"Knappkonfiguration",
+	"23":	"Ändra språk",
+	"24":	"Volym",
+	"25":	"Menyljud",
+	"26":	"Ljusstyrka",
+	"27":	"Ladda om roms",
+	"28":	"Fabriksinställningar",
+	"29":	"Flashningsläge",
+	"30":	"Om enhet",
+	"31":	"Knapptest",
+	"32":	"ADC-Info",
+	"33":	"Status:",
+	"34":	"Fördröjning:",
+	"35":	"OK-frekvens:",
+	"36":	"Felfrekvens:",
+	"37":	"STOPPAD",
+	"38":	"KÖR",
+	"39":	"ADC:",
+	"40":	"Laddar",
+	"41":	"OTG",
+	"42":	"Frånkopplad",
+	"43":	"Error",
+	"44":	"Tillagd i favoriter",
+	"45":	"AVBRYT",
+	"46":	"OK",
+	"47":	"sekunder",
+	"48":	"i",
+	"49":	"Spellista",
+	"50":	"Rensa lista",
+	"51":	"Starta",
+	"52":	"Ta bort från lista",
+	"53":	"Töm alla spelcachar",
+	"54":	"Ladda om alla roms",
+	"55":	"Lägg till i favoriter", 
+	"56":	"Uppgradera",
+	"57":	"Uppdatera systemet?",
+	"58":	"SD-utrymme lågt\n%dM ledigt",
+	"59":	"Modelnummmer",
+	"60":	"Serienummer",
+	"61":	"Firmware-version",
+	"62":	"CPU-frekvens",
+	"63":	"%dMHz",
+	"64":	"Minnesstorlek",
+	"65":	"%dMB",
+	"66":	"Använt/totalt utrymme",
+	"67":	"Nollställ inställningar",
+	"68":	"Nollställ alla systeminställningar?\nSD-kortets innehåll\ntas ej bort.",
+	"69":	"Läs om roms på SD-kortet",
+	"70":	"Nollställ",
+	"71":	"standard",
+	"72":	"90°",
+	"73":	"map3",
+	"74":	"okänt",
+	"75":	"LADDAR",
+	"76":	"Laddar",
+	"77":	"Rensa senastelistan?",
+	"78":	"Note: SD-kortets innehåll\ntas ej bort.",
+	"79":	"Rensa favoritlistan?",
+	"80":	"Note: SD-kortets innehåll\ntas ej bort.",
+	"81":	"Remove from favorites list?",
+	"82":	"Note: SD-kortets innehåll\ntas ej bort.",
+	"83":	"Skannar...",
+	"84":	"SPELINFO",
+	"85":	"Skapa cache",
+	"86":	"Stäng av",
+	"87":	"Stänger av om %d sekunder...",
+	"88":	"VÄLJ",
+	"89":	"TILLBAKA",
+	"90":	"NAV",
+	"91":	"MENY",
+	"92":	"Återuppta",
+	"93":	"Spara",
+	"94":	"Ladda",
+	"95":	"Volym",
+	"96":	"Lämna till menyn",
+	"97":	"Tillstånd %d/%d",
+	"98":	"Inget tillstånd",
+	"99":	"Ingen miniatyr",
+	"100":	"Laddning misslyckades",
+	"101":	"Sparning misslyckades",
+	"102":	"Tillstånd %d sparat",
+	"103":  "Fullskärm",
+	"104":  "PÅ",
+	"105":  "AV",
+	"106":  "Avancerat",
+	"107":  "Appar",
+	"108":  "Är du säker att du vill\nstänga av din enhet?",
+    "109":  "Kanpptest",
+    "110":  "Knappkonfiguration",
+    "111":  "STÄNG AV",
+    "112":  "SPARA OCH STÄNG AV",
+    "113":  "%d minuter",
+    "114":  "Av",
+	"115":  "Sömntimer",
+	"116":  "Ta bort spel från SD-kort?",
+    "117":  "Bekräfta att du vill ta bort\ndet här spelet från SD-kortet.",
+    "118":  "Skärm",
+    "119":  "Ljusstyrka",
+    "120":  "Färg",
+    "121":  "Färgmättnad",
+    "122":  "Kontrast",
+	"123":  " \n \nSkärminställningar\nsparade.\n \n ",
```

Comment:
Can you add a note, asking the user to reboot the device (new for V4.1)

Reply:
Done!